core/src/main/kotlin/gui/CheckersScreen.kt

```diff
@@ -33,19 +32,19 @@ class CheckersScreen(val game: Game,
     val guiPlayer = GUIPlayer(colorPlayer)
     private lateinit var gameModel: CheckersGUIGame
 
-    private val newGameButton = Button(Texture(GUIConstants.newGameButtonUpPath),
-            Texture(GUIConstants.newGameButtonDownPath),
-            GUIConstants.newGameButtonXS, GUIConstants.newGameButtonYS,
-            GUIConstants.newGameButtonWidthS, GUIConstants.newGameButtonHeightS)
-
-    private val exitButton = Button(Texture(GUIConstants.exitButtonUpPath),
-            Texture(GUIConstants.exitButtonDownPath),
-            GUIConstants.exitButtonXS, GUIConstants.exitButtonYS,
-            GUIConstants.exitButtonWidthS, GUIConstants.exitButtonHeightS)
-    private val menuButton = Button(Texture(GUIConstants.menuButtonUpPath),
-            Texture(GUIConstants.menuButtonDownPath),
-            GUIConstants.menuButtonXS, GUIConstants.menuButtonYS,
-            GUIConstants.menuButtonWidthS, GUIConstants.menuButtonHeightS)
+    private val newGameButton = Button(Texture(GUIConstants.newGameButton.upPath),
```

Comment:
Now you can define a function that gets ButtonSettings and returns Button (maybe a constructor of Button)

Reply:
In some places I leave old constructors because they use special fields (with -s or -se suffixes, etc).